conf/pipeline/sarek/munin.config

```diff
@@ -0,0 +1,16 @@
+//Profile config names for nf-core/configs
+
+params {
+  config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
+  config_profile_description = 'sarek MUNIN profile provided by nf-core/configs.'
```

Comment:
Presumably this overwrites the param set in the main config? https://github.com/nf-core/configs/blob/a283c5b7c4b491712e610dde9ce79f5578aac07f/conf/munin.config#L3

I guess this is fine.. Maybe not necessary though.

Reply:
Yes, the second config file overwrite all the params from the first one.
It would be better to append actually in that case.

Reply:
I do think it's good to keep that, that way you know that the main institute profile has been overwritten by a pipeline/institute config

Reply:
Not necessarily completely overwritten though. Most of the time it will be adding additional stuff I imagine.

Reply:
Existing params are overwritten

Reply:
But yeah, I do think it's good to know that this is more than the nf-core/configs classic institutional profile